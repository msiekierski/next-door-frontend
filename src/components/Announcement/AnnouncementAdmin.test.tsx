import React from "react";
import renderer from "react-test-renderer";
import AnnouncementAdmin from "./AnnoucementAdmin"
import IAnnouncementAdmin from "./IAnnoucementAdmin";

const props: IAnnouncementAdmin = {
    idAnnouncement: 1,
    idAccount: 1,
    announcementType: "administrative",
    title: "Ann",
    desc: "desc",
    creationDate: new Date().toString(),
    author: "John Doe",
    comments: []
}

it("renders correctly", () => {
  const tree = renderer.create(<AnnouncementAdmin {...props} />);
  expect(tree).toMatchSnapshot();
});
