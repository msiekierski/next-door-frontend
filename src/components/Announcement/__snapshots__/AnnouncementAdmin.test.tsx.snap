// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-3 card"
>
  <div
    className="bg-warning card-header"
  />
  <div
    className="card-body"
  >
    <div
      className="d-flex justify-content-between card-title h5"
    >
      <span>
        Ann
      </span>
      <div
        className="flex-column"
      >
        <div
          className="text-right mb-3 card-subtitle h6"
        />
        <div
          className="text-muted text-right card-subtitle h6"
        >
          John Doe
        </div>
      </div>
    </div>
    <p
      className="card-text"
    >
      desc
    </p>
    <div
      className="text-muted text-right card-subtitle h6"
    >
      2021-1-22
      <br />
      01:31 AM
    </div>
  </div>
  <div
    className="d-flex justify-content-between card-footer"
  >
    <a
      className="card-link"
      href=""
      onClick={[Function]}
    >
      Reply
    </a>
    <a
      className="card-link"
      href=""
      onClick={[Function]}
    >
      No replies yet
    </a>
  </div>
</div>
`;
